admin/catalog.php

```diff
@@ -125,12 +125,12 @@
             foreach ($songs as $song_id) {
                 Song::update_enabled(true, $song_id);
             }
-            $body = count($songs) . nT_(' Song Enabled', ' Songs Enabled', count($songs));
+            $body = count($songs) . ' ' . nT_('Song Enabled', 'Songs Enabled', count($songs));
         } else {
-            $body = T_('No Disabled Songs selected');
+            $body = T_('No disabled songs selected');
```

Comment:
Why writing "disabled songs" lower case when you mostly write the stuff, that's the current subject, uppercase?